vendor/vendor.json

```diff
@@ -0,0 +1,268 @@
+{
+	"comment": "",
+	"ignore": "test",
+	"package": [
+		{
+			"checksumSHA1": "E2a5TaCbwqs7PwP1fvfWLhvgr38=",
+			"comment": "v2.2.0-rc.0-72-g106d918",
+			"path": "github.com/coreos/etcd/Godeps/_workspace/src/github.com/coreos/go-systemd/journal",
```

Comment:
You don't see it but it's needed by the etcd code.

I ran `govendor migrante godep` and verified packages and related commit hashes. The code wouldn't compile because these were missing but probably it was something with `govendor`?